clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -1356,7 +1356,11 @@ struct ConvertCIRToLLVMPass
 
   void processCIRAttrs(mlir::ModuleOp moduleOp);
 
-  virtual StringRef getArgument() const override { return "cir-flat-to-llvm"; }
+  StringRef getDescription() const override {
+    return "Convert the prepared CIR dialect module to LLVM dialect";
```

Comment:
Can we get that for free by adding `let description` for these passes?

Reply:
I do not think these passes are described in a TableGen file.
Do you mean we should define them in a TableGen file first and then add such a description there?